### PR TITLE
Fix rhea system overview

### PR DESCRIPTION
**Pull request by porgull**
partially addresses #14

### CODE REVIEW

File: systems/includes/rhea-compute-nodes.txt [hunk 1]
```diff
@@ -14,16 +14,16 @@ Rhea contains 521 compute nodes separated into two partitions:
```
Comment (grahamlopez) [inline systems/includes/rhea-compute-nodes.txt]: I don't really see any reason to use a separate file for this section. Let's just go ahead and inline this in the right spot in the user guide.

File: systems/includes/rhea-compute-nodes.txt [hunk 2]
```diff
@@ -35,7 +35,7 @@ With Hyper-Threading enabled, these nodes have 56 logical cores that can execute
 
 Please see the :ref:`batch-queues-on-rhea` section to learn about the queuing
 policies for these two partitions. Both compute partitions are accessible
-through the same batch queue from Rhea’s login nodes.
+through the same batch queue from Rhea’s `login nodes <#overview>`__.
```
Comment (grahamlopez) [inline systems/includes/rhea-compute-nodes.txt]: can we make this a cross-ref?

Comment (porgull) [reply systems/includes/rhea-compute-nodes.txt]: yep, committed

File: systems/includes/rhea-compute-nodes.txt (hunk 1 above)
Comment (porgull) [reply systems/includes/rhea-compute-nodes.txt]: it is now in-line

**Comment by porgull** [comment]
@grahamlopez the links to filesystems don't work in this PR, but those are tentative, so should that be handled in a separate issue?

**Comment by grahamlopez** [comment]
@porgull yes, we will have to fix those later.

**Pull request merged**